src/TestCases/Traits/AssertTrait.php

```diff
@@ -73,14 +76,16 @@ public static function assertConstraints(string $expected, ConstraintViolationLi
      * phpcs:disable
      *
      * @param array<string, array<string>> $expectedValues
-     * @param \LoyaltyCorp\Search\Interfaces\ClientInterface $client
+     * @param \Eonx\TestUtils\Helpers\Interfaces\ClientStubInterface|null $client
      *
      * @return void
      *
      * phpcs:enable
      */
-    public static function assertDocumentIdsUpdated(array $expectedValues, ClientInterface $client): void
+    public static function assertDocumentIdsUpdated(array $expectedValues, ?ClientStubInterface $client = null): void
```

Comment:
I dont think a default nullable makes sense here - if you dont have a Search client you cant do assertions against it